homeassistant/components/fritz/common.py

```diff
@@ -325,27 +326,33 @@ async def async_scan_devices(self, now: datetime | None = None) -> None:
         """Wrap up FritzboxTools class scan."""
         await self.hass.async_add_executor_job(self.scan_devices, now)
 
+    def manage_device_info(
+        self, dev_info: Device, dev_mac: str, consider_home: bool
+    ) -> bool:
+        """Update device lists."""
+        _LOGGER.debug("Client dev_info: %s", dev_info)
+
+        if dev_mac in self._devices:
+            self._devices[dev_mac].update(dev_info, consider_home)
+            return False
+
+        device = FritzDevice(dev_mac, dev_info.name)
+        device.update(dev_info, consider_home)
+        self._devices[dev_mac] = device
+        return True
+
+    def signal(self, new_device: bool) -> None:
+        """Signal device."""
```

Comment:
Would suggest to name this method a bit more precise
```suggestion
    def send_signal_device_update(self, new_device: bool) -> None:
        """Signal device data updated."""
```